app/routes/ready-reckoner/land-values.js

```diff
@@ -81,9 +96,13 @@ module.exports = [
         return h.view(pageDetails.template, pageContent)
       }
 
-      const body = addState(payload)
+      const standardsSet = {
```

Comment:
I can't think of a better name than `standardsSet` for the collection of the standards and associated properties. Suggestions welcome.

Reply:
Did we decide on `scheme` in the chat on Friday?